topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: tutorial_hands_on
+
+title: "Pulsar usage on SURF Research Cloud"
+zenodo_link: ""
+questions:
+  - How do I start a Pulsar instance on SURF Research Cloud?
+  - How do I connect to Pulsar?
+objectives:
+  - Be able to attach a Pulsar node to Galaxy and use interactive tools
+requirements:
+  - type: "none"
+    title: Access to the SURF Research Cloud
+  - type: "none"
+    title: SRAM authentication
+  - type: "none"
+    title: An SSH key connected to your account
+  - type: "none"
+    title: Have a basic understanding of how Galaxy works
+time_estimation: "30m"
+level: Introductory
+contributions:
+  authorship:
+  - hexylena
+  - mirelaminkova
+  funding:
+  - surf
+
+edam_ontology:
+- topic_0605 # Informatics
+- topic_3071 # Data Management
+
+abbreviations:
+  SRC: SURF Research Cloud
+  GDPR: General Data Protection Regulations
+  SRAM: SURF Research Access Management
+  CO: Collaborative Organisation
+
+follow_up_training:
+  - type: "internal"
+    topic_name: admin
+    tutorials:
+      - surf-research-cloud-galaxy
+---
```

Comment:
Should these tutorials be listed in the "Running Galaxy on the Cloud" section of the GTN admin topic page? 

If so, please add `subtopic: cloud` to the metadata (now they are placed at bottom in the 'other' section)